### DISCUSSION TITLE
ibax şu ana kadar çıkan projelerin içinde en kapsamlısı ve sağlam proje

### DISCUSSION THREAD

**Discussion by umit65** [IBAX-io/go-ibax#584] [Ecolibs]
It will be the best alternative in a project that will not come out as much as those who want ibax, and it will still be a solid project. Can there be any claim to the contrary?

**Comment by ozgurizmir** [comment]
haklısınız

**Comment by umit65** [comment]
ibax cok saglam bir proje en iyi alternatifi yine kendisi

**Reply by umit65** [reply]
Yani ibax in en iyi alternatifi yine kendisi

**Reply by umit65** [reply]
Bencede tartismasiz ibax sence?
